src/gc/gcee.cpp

```diff
@@ -543,7 +540,7 @@ void GCProfileWalkHeap()
     }
 #endif // defined (GC_PROFILING)
 
-#ifdef FEATURE_EVENT_TRACE
+#if  defined (GC_PROFILING)//UNIXTODO: Enable this for FEATURE_EVENT_TRACE
```

Comment:
No the TODO comment is important as it preserves the idea of "what should have happened" when next person is going to fix it